flasksrc/schema.sql

```diff
@@ -22,3 +26,31 @@ CREATE TABLE toynet_value_inspirations (
   quote TEXT NOT NULL,
   FOREIGN KEY (value_id) REFERENCES toynet_value (id)
 );
+
+
+-- quizzes submodule
+
+CREATE TABLE toynet_quizzes (
+  quiz_id INTEGER PRIMARY KEY,
+  created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE toynet_quiz_questions (
+  quiz_id INTEGER NOT NULL,
+  question_id  INTEGER NOT NULL,
+  question TEXT NOT NULL,
+  answer INTEGER NOT NULL,
+  PRIMARY KEY (quiz_id, question_id)
+  FOREIGN KEY (quiz_id) REFERENCES toynet_quiz (quiz_id)
+);
+
+CREATE TABLE toynet_quiz_options (
+  quiz_id INTEGER NOT NULL,
+  question_id INTEGER NOT NULL,
+  option_id INTEGER NOT NULL,
+  option TEXT NOT NULL,
+  PRIMARY KEY (quiz_id, question_id, option_id)
+  FOREIGN KEY (quiz_id) REFERENCES toynet_quiz (quiz_id)
+  FOREIGN KEY (question_id) REFERENCES toynet_quiz_questions (question_id)
```

Comment:
These tables look great! Can you revise this `FOREIGN KEY` setup based on the following document? I believe they should be grouped.
https://www.ibm.com/support/knowledgecenter/SSGU8G_14.1.0/com.ibm.sqls.doc/ids_sqs_0535.htm